dev-python/et_xmlfile/et_xmlfile-1.1.0.ebuild

```diff
@@ -3,18 +3,18 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
 
 inherit distutils-r1
 
 MY_PV=${PV%.0}
 MY_P=${PN}-${MY_PV}
 
 DESCRIPTION="An implementation of lxml.xmlfile for the standard library"
-HOMEPAGE="https://pypi.org/project/et-xmlfile/"
+HOMEPAGE="https://pypi.org/project/et-xmlfile/ https://foss.heptapod.net/openpyxl/et_xmlfile"
 SRC_URI="
 	https://foss.heptapod.net/openpyxl/et_xmlfile/-/archive/${MY_PV}/${MY_P}.tar.gz"
-S=${WORKDIR}/${MY_P}
+S="${WORKDIR}/${MY_P}"
```

Comment:
Normally we don't really quote `S` since it's not supposed to contain spaces. I'll revert this line.

Reply:
Noted, thank you.